src/network/taraxa_capability.cpp

```diff
@@ -514,7 +514,15 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
                                      << peer_pbft_round << " previous round next votes size "
                                      << peer_pbft_previous_round_next_votes_size
                                      << ". Will send out bundle of next votes";
-        sendPbftNextVotes(_nodeID);
+
+        auto next_votes_bundle = next_votes_mgr_->getNextVotes();
```

Comment:
Why is this code here and not in sendPbftNextVotes?

Reply:
Move to broadcastPreviousRoundNextVotesBundle. Because sending each peer will call sendPbftNextVotes, this will only call sendPbftNextVotes one time for all peers

Reply:
Ok